contents/blog/posthog-vs-growthbook.md

```diff
@@ -0,0 +1,637 @@
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td>Target users based on user properties, custom contexts</td>
+        </tr>
+       <tr>
+            <td><strong>JSON payloads</strong></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td>Flags return JSON</td>
+        </tr>
+       <tr>
+            <td><strong>Environments</strong></td>
+            <td className="text-center">Partial</td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td>Manage flags for dev, stage, prod</td>
+        </tr>
+       <tr>
+            <td><strong>Scheduling</strong></td>
+            <td className="text-center"><span className="text-red text-lg">✖</span></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td>Schedule flags to turn on or off</td>
+        </tr>
+       <tr>
+            <td><strong>Early access management</strong></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td className="text-center"><span className="text-red text-lg">✖</span></td>
+            <td>Manage betas, test features</td>
+        </tr>
+       <tr>
+            <td><strong>Bootstrapping</strong></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td className="text-center"><span className="text-red text-lg">✖</span></td>
+            <td>Flags available on frontend application load</td>
+        </tr>
+    </tbody>
+</table>
+</div>
+
+- **Targeting:** GrowthBooks custom targeting with attributes needs to be set every session and defined in-app before use. It is also only available on the paid Pro plan. PostHog automatically sets its equivalent ([properties](/docs/getting-started/user-properties)) on users. You don’t need to pre-defined them and unlimited custom values are free.
+
+- **[Bootstrapping](/docs/feature-flags/bootstrapping):** PostHog’s JavaScript web SDK enables you to pass flags directly from the backend before the app loads. This makes sure they are available right away and prevents flickering. GrowthBook instead recommends moving the A/B test or flag logic earlier in the page load (server-side) to prevent this.
+
+## Experimentation
+
+Experimentation is where PostHog and GrowthBook’s functionality diverges. Both enable you to create [A/B/n tests](/tutorials/abn-testing) with custom goals and calculate statistical significance. The big difference between them is:
+
+- GrowthBook’s A/B testing is statistically focused and deeper. It offers frequentist This does require writing SQL queries for integrations with event trackers as well as goals and analysis. It also requires more statistics knowledge to take advantage of.
```

Comment:
Second sentence doesn't seem to make sense, and I don't know what 'statistically focused' means -- would be good to spell this out more.

Reply:
Rewrote